src/consensus/raft/test/driver.h

```diff
@@ -221,7 +221,9 @@ class RaftDriver
   {
     std::cout << "  KV" << node_id << "->>Node" << node_id
               << ": replicate idx: " << idx << std::endl;
-    _nodes.at(node_id).raft->replicate({{idx, data, true}});
+    std::vector<std::tuple<kv::Version, std::vector<uint8_t>, bool>> batch;
```

Comment:
Batch batch? I would be nice to share the using defined below here too (perhaps pull them in a common header if they aren't in raft.h already?)

Reply:
kvtypes.h is probably the best place to put `Batch` since it is used in the kv

Reply:
using `BatchVector` and `BatchDetachedBuffer` defined in kvtypes.h. Used in raft, pbft, kv, and in the tests.